gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectAttributes.java

```diff
@@ -131,25 +145,27 @@ public FileTime lastAccessTime() {
 
   @Override
   public Object fileKey() {
-    return metadata.getFilename();
+    return name;
```

Comment:
OK, thanks, but what's wrong with `name`? It seems to work just as well given that we don't need to be able to go from fileKey back to a blob.